infra/src/Pos/Infra/InjectFail.hs

```diff
@@ -71,14 +71,18 @@ data FInjects m     = FInjects
     , listFInjects   :: m [FInject]
     }
 
+logWarningLoud :: (CanLog m, HasLoggerName m) => String -> m ()
```

Comment:
Waiting for the next patch `logWarningLouder` :trollface:

Reply:
: -)